crypto/rand/rand_lib.c

```diff
@@ -25,6 +25,39 @@
 /* Macro to convert two thirty two bit values into a sixty four bit one */
 #define TWO32TO64(a, b) ((((uint64_t)(a)) << 32) + (b))
 
+/*
+ * Check for the existence and support of POSIX timers.  The standard
+ * says that the _POSIX_TIMERS macro will have a positive value if they
+ * are available.
+ *
+ * However, we want an additional constraint: that the timer support does
+ * not require an extra library dependency.  Early versions of glibc
+ * require -lrt to be specified on the link line to access the timers,
+ * so this needs to be checked for.
+ *
+ * It is worse because some libraries define __GLIB__ but don't
+ * support the version testing macro.  (e.g. uClibc and OS/X).  This means
```

Comment:
The macro is `__GLIBC__`, not `__GLIB__`. And I can *not* confirm that MacOS X defines it. Well, at least XCode toolchain, i.e. official SDK doesn't, and we don't support anything else.

Reply:
It was the undefined macro on OSX that caused the problem not GLIBC.
I've fixed the comment.

I don't know why I kept typing glib not glibc.  Addressed now.